include/network.h

```diff
@@ -25,23 +25,55 @@
 //    Network related functions and definitons
 //
 //---------------------------------------------------------------------------
+#pragma once
 
 #include "remotecontrol.h"
 #include "socketserver.h"
 #include "ntptimeclient.h"
 #include "secrets.h"                          // copy include/secrets.example.h to include/secrets.h
 
-extern uint8_t g_Brightness;
-extern bool g_bUpdateStarted;
-extern WiFiUDP g_Udp;
-#if INCOMING_WIFI_ENABLED
-    extern SocketServer g_SocketServer;
-#endif
-void processRemoteDebugCmd();
+#if ENABLE_WIFI
+    extern uint8_t g_Brightness;
+    extern bool g_bUpdateStarted;
+    extern WiFiUDP g_Udp;
+    #if INCOMING_WIFI_ENABLED
+        extern SocketServer g_SocketServer;

```

Comment:
Taking a look around the code-base there is a mix of indent includes/extern, all other blocks seem to follow indent so went with that.